### ISSUE TITLE
Missing fields on Itbit `fetchMarkets` model

### ISSUE THREAD

**Issue by fbadiola** [ccxt/ccxt#6552]
- OS: Linux
- Programming Language version: JavaScript
- CCXT version: 1.23.30
- Exchange: itBit
- Method: fetchMarkets

fetchMarkets model should be something like this:

```javascript
[
//...
{
  id: 'BCHSV-BTC',
  symbol: 'BSV/BTC',
  baseId: 'BCHSV',
  quoteId: 'BTC',
  base: 'BSV',
  quote: 'BTC',
  active: true,
  precision: { amount: 8, price: 6 },
  limits: {
    amount: { min: 0.001, max: 10000000000 },
    price: { min: 0.000001, max: 99999999000 },
    cost: { min: 0.00001, max: 99999999 }
  },
 info: {
  // ...
 }
},
//...
]
```

But itBit fetchMarkets schema is this one:

```javascript
[
  //...
  {
    limits: {
      amount: {},
      price: {},
      cost: {}
    },
    precision: {},
    tierBased: undefined,
    percentage: undefined,
    taker: 0.0034999999999999996,
    maker: -0.0003,
    id: 'ETHSGD',
    symbol: 'ETH/SGD',
    base: 'ETH',
    quote: 'SGD'
  },
  // ...
]
```

I miss `baseId` and `quoteId` fields. Is it right?

Thanks.

**Comment by fbadiola** [comment]
There're other fields that doesn't exists in the other model. Are there a way to normalize models to avoid know each case?

**Comment by kroitor** [comment]
> There're other fields that doesn't exists in the other model. Are there a way to normalize models to avoid know each case?

For `itbit` those markets are hardcoded here:

- https://github.com/ccxt/ccxt/blob/master/js/itbit.js#L63

This is because itbit does not serve them from the API.

I will add those fields there soon.

> Are there a way to normalize models to avoid know each case?

We have the tests for it, will look into those shortly.

**Comment by kroitor** [comment]
@fbadiola this should be fixed in 1.23.31+, let me know if you find more. Thank you!
